### PR TITLE
[typo](docs)Modification instructions and examples for adding schema change key columns

### PR DESCRIPTION
**Pull request by hf200012**
Modification instructions and examples for adding schema change key columns

# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [ ] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [ ] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [ ] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/zh-CN/docs/advanced/alter-table/schema-change.md [hunk 1]
```diff
@@ -181,6 +181,42 @@ ADD COLUMN k5 INT default "1" to rollup2;
 
 同时，不允许向 Rollup 中加入 Base 表已经存在的列。如果用户需要这样做，可以重新建立一个包含新增列的 Rollup，之后再删除原 Rollup。
 
+### 修改 Key 列
+
+修改表的 Key 列是通过 `key` 关键字完成，下面我们通过一个例子来看。
```
Comment (morningman) [inline docs/zh-CN/docs/advanced/alter-table/schema-change.md:186]: 这个用法只针对 duplicate key 表的key列，即排序列的修改。其他data model 的表不需要这个写法。

**Comment by chenlinzhong** [comment]
I suggest adding the command "show alter table column" , so that user can view the progress

File: docs/zh-CN/docs/advanced/alter-table/schema-change.md (hunk 1 above)
Comment (hf200012) [reply docs/zh-CN/docs/advanced/alter-table/schema-change.md:186]: solve

**Comment by hf200012** [comment]
> I suggest adding the command "show alter table column" , so that user can view the progress

ok

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**